pkg/ovirt/components/VmOverviewColumn.jsx

```diff
@@ -70,6 +71,7 @@ const VmOverviewColumn = ({ vm, providerState }) => { // For reference, extend i
                     <table className='form-table-ct'>
                         <VmProperty title={_("Description:")} value={clusterVm.description} id={`${idPrefix}-description`} />
                         <VmProperty title={_("Address:")} value={clusterVm.fqdn} id={`${idPrefix}-fqdn`} />
+                        <VmProperty title={_("Running since:")} value={formatDateTime(clusterVm.startTime)} id={`${idPrefix}-starttime`} />
```

Comment:
Everything else is Headline Case, so this should use that too, ie. "Running Since"
https://www.patternfly.org/styles/terminology-and-wording/#capitalization

Reply:
done